src/diff.ts

```diff
@@ -0,0 +1,43 @@
+import * as core from '@actions/core';
+import { Repo } from './github';
+import { VersionResponse } from 'bump-cli';
+import { bumpDiffComment, shaDigest } from './common';
+
+interface VersionWithDiff extends VersionResponse {
+  diff_summary: string;
+  diff_public_url: string;
+}
+
+const commentTitle = '🤖 API change detected:';
+const codeBlock = '```';
+const poweredByBump = '> _Powered by [Bump](https://bump.sh)_';
+
+export async function run(version: VersionResponse): Promise<void> {
+  if (!isVersionWithDiff(version)) {
+    core.info('No diff found, nothing more to do.');
+    return;
+  }
+
+  const repo = new Repo();
+  const digest = shaDigest(version.diff_summary);
+  const body = buildCommentBody(version, digest);
+
+  await repo.createOrUpdateComment(body, digest);
+}
+
+function isVersionWithDiff(version: VersionResponse): version is VersionWithDiff {
+  return version.diff_summary !== undefined;
+}
+
+function buildCommentBody(version: VersionWithDiff, digest: string) {
+  return [commentTitle]
+    .concat([codeBlock, version.diff_summary, codeBlock])
+    .concat([viewDiffLink(version), poweredByBump, bumpDiffComment(version.id, digest)])
+    .join('\n');
+}
+
+function viewDiffLink(version: VersionWithDiff): string {
+  return `
+[View visual diff](${version.diff_public_url})
```

Comment:
<img width="539" alt="Capture d’écran 2021-06-30 à 09 49 46" src="https://user-images.githubusercontent.com/30788/123922628-8419e000-d988-11eb-8979-37f29285afa6.png">

Sorry for the ping/pong, but finally I think that we should be consistent with our changelog UI, and use the `View documentation diff` here. WDYT?

Reply:
:+1: